packages/inertia/src/mergeQueryStringsWithData.js

```diff
@@ -0,0 +1,10 @@
+import qs from 'qs'
+import deepmerge from 'deepmerge'
+
+export default function mergeQueryStringsWithData(method, url, data) {
```

Comment:
`?foo=bar&baz=qux` is a (singular) query string.

```suggestion
export default function mergeQueryStringWithData(method, url, data) {
```

I believe singular is a better method name, otherwise it sounds like it accepts multiple query strings.

Reply:
Great point, will update. 👍